README.md

```diff
@@ -7,59 +7,411 @@
+    }
+  },
+
+  // Even though this function is designed for creating a claim, we can
+  // very easily repurpose it to make a payment for settlement.
+  createOutgoingClaim: async function (ctx, outgoingBalance) {
+    // If a new max value is set, the maxValueTracker returns the previous max
+    // value. We tell the maxValueTracker that we're gonna pay the entire
+    // outgoingBalance we owe, and then look at the difference between the last
+    // balance and the outgoingBalance to determine how much to pay.
+    // If we've already paid out more than outgoingBalance, then it won't be the
+    // max value. The maxValueTracker will return outgoingBalance as the result,
+    // and outgoingBalance - outgoingBalance is 0. Therefore, we send no payment.
+    const lastPaid = ctx.state.amountSettled.setIfMax({ value: outgoingBalance, data: null })
+    const diff = outgoingBalance - lastPaid.value
+
+    if (!diff) {
+      return
+    }
+
+    // We take the transaction ID from the payment we send, and give it as an
+    // identifier so out peer can look it up on the network and verify that we
+    // paid them. Another approach could be to return nothing from this
+    // function, and have the peer automatically track all incoming payments
+    // they're notified of on the settlement ledger.
+    const txid = Network.makePaymentToPeer(diff)
+
+    return { txid }
+  },
+
+  handleIncomingClaim: async function (ctx, claim) {
+    const { txid } = claim
+    const payment = await Network.getPayment(txid)
+
+    if (!payment) {
+      return
+    }
+
+    // It doesn't really matter whether this is fulfilled or not, we just need
+    // it to affect the incoming balance so we know how much has been received.
+    // We use the txid as the ID of the incoming payment, so it's impossible to
+    // apply the same incoming settlement transfer twice.
+    await ctx.state.incomingSettlements.prepare({
+      id: txid,
+      amount: payment.amount
+    }, true) // isIncoming: true
+  }
+
+}
 ```
+
+## Backend API, with Extensions for Payment Channels
+
+-------
+
+### `async TransferLog.getIncomingFulfilled ()`
+
+Get the sum of all incoming payments in the `fulfilled` state.
+
+#### Returns
+
+- `return` (Integer String) incoming balance.
+
+-------
+
+### `async TransferLog.getIncomingFulfilledAndPrepared ()`
+
+Get the sum of all incoming payments, including those which are in the `prepared` state.
+
+#### Returns
+
+- `return` (Integer String) highest incoming balance.
+
+-------
+
+### `async TransferLog.getOutgoingFulfilled ()`
+
+Get the sum of all outgoing payments in the `fulfilled` state.
+
+#### Returns
+
+- `return` (Integer String) outgoing balance.
+
+-------
+
+### `async TransferLog.getOutgoingFulfilledAndPrepared ()`
+
+Get the sum of all outgoing payments, including those which are in the `prepared` state.
+
+#### Returns
+
+- `return` (Integer String) highest outgoing balance.
+
+-------
+
+### `getMaxValueTracker (key)`
```

Comment:
`getMaxValueTracker` and `getTransferLog` should be the first functions documented. It's confusing that they're buried underneath the definition of the TransferLog type.